src/redis-cli.c

```diff
@@ -6314,8 +6325,8 @@ static int clusterManagerCommandCreate(int argc, char **argv) {
                 continue;
             }
             redisReply *reply = NULL;
-            reply = CLUSTER_MANAGER_COMMAND(node, "cluster meet %s %d",
-                                            first->ip, first->port);
+            reply = CLUSTER_MANAGER_COMMAND(node, "cluster meet %s %d %d",
+                                            first->ip, first->port, first->bus_port);
```

Comment:
The CLUSTER MEET bus-port parameter was added in Redis 4.0.0. With this change, redis-cli no longer works for Redis 3.x clusters. I think it is OK. Just pointing it out.

Reply:
yes, keep it, do it in the last commit